### ISSUE TITLE
When editing a post to a channel bbcode url is doubled.

### ISSUE THREAD

**Issue by jeroenpraat** [redmatrix/redmatrix#62]
_From @Boerny on November 12, 2013 15:25_

After editing a post the entry of a [url]https://bug.com[/url] was doubled.
and looked like this
[url=http://https://bug.com]http://https://bug.com[/url]
in the post.

_Copied from original issue: friendica/red#197_

**Comment by jeroenpraat** [comment]
_From @friendica on November 12, 2013 23:17_

Unfortunately, I've tried this with and without the visual editor and can't duplicate it. We probably need a clear sequence of steps to reproduce. (Yes, it seems easy enough, but I tried a bunch of things and got the correct result every time).

**Comment by jeroenpraat** [comment]
_From @Boerny on November 13, 2013 10:20_

After trying several times, I was not able to reproduce it either.

**Comment by jeroenpraat** [comment]
_From @friendica on November 15, 2013 0:46_

I think I figured it out. The addition of http:// to the URL is the clue. Also this happened when you made a comment. The bbcode formatter for URLs in comments puts "http://" in between the tags by default. I'm at least 60% confident this is what triggered the sequence of events -  you probably didn't notice the extra text being added to the link. With two http(s):// fragments in the link URL this would throw a tizzy to the bbcode parser which subsequently turned this into a botched nightmare. As I've seen the conversation which led to this issue I've got a good idea of the sequence of steps you went through and this fits the pattern.

Anyway, I've thought about it and the "http://" just seems to always get in the way. Most pasted links include the scheme - and then you've got to take an extra step to remove the redundant bits. 

So I've removed this placeholder text from the code. It's more trouble than it's worth.
